website/static/js/tests/registrationEditorExtensions.test.js

```diff
@@ -0,0 +1,82 @@
+/*global describe, it, expect, example, before, after, beforeEach, afterEach, mocha, sinon*/
+'use strict';
+var assert = require('chai').assert;
+var Raven = require('raven-js');
+var $ = require('jquery');
+var RegistrationEditor = require('js/registrationEditorExtensions');
+
+// Add sinon asserts to chai.assert, so we can do assert.calledWith instead of sinon.assert.calledWith
+sinon.assert.expose(assert, {prefix: ''});
+
+
+describe('RegistrationEditor.osfUploader', () => {
+    var itemIsOsfstorage = {
```

Comment:
TODO: Touch base on where uploader is used in this PR (none of the existing registration types seem to support file uploads)